cmd/mrcalc.cpp

```diff
@@ -984,15 +982,14 @@ class OpTernary : public OpBase {
 void run() {
   std::vector<StackEntry> stack;
 
-  for (int n = 1; n < App::argc; ++n) {
-
-    const Option *opt = match_option(App::argv[n]);
+  for (const auto &argument : App::raw_arguments_list) {
+    const Option *opt = match_option(argument);
     if (opt) {
 
       if (opt->is("datatype"))
-        ++n;
```

Comment:
These increments on `n` can't be removed. If a standard option is not just a flag, but receives as input one or more arguments, then it's not just the appearance of the command-line option that needs to be skipped in construction of the operand stack, the arguments attributed to that option need to be skipped also.

Reply:
Worth adding a `mrcalc` test that intersperses uses of the standard options between operands.